tests/test_utilities.py

```diff
@@ -135,6 +135,17 @@ def test_default_to(case, expected):
     assert _.default_to(*case) == expected
 
 
+@parametrize('case,expected', [
+    (([1, 10, 20]), 1),
+    (([None, 10, 20]), 10),
+    (([None, None, 20]), 20),
+    (([None, [1, 2], [3, 4]]), [1, 2]),
+    (([None, None, [3, 4]]), [3, 4])
+])
```

Comment:
Can add a case for all `None` values as input just to verify `None` is returned in that case.